modules/local/epytope_peptide_prediction.nf

```diff
@@ -9,6 +9,7 @@ process EPYTOPE_PEPTIDE_PREDICTION {
     input:
     tuple val(meta), path(splitted), path(software_versions)
     val netmhc_paths
+    path(expression)
```

Comment:
We could do it with the rnaseq file similar than with the optional netmhc_paths, meaning declaring it as a val instead of a path.